bears/css/StyleLintBear.py

```diff
@@ -0,0 +1,32 @@
+import os
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylelint',
+        output_format='regex',
+        output_regex=r'.+\n\s(?P<line>\d+):(?P<column>\d+)'
+                     r'\s+✖\s(?P<message>.+)')
```

Comment:
you can map the error sign(the cross), the warning sign...... with ``(\D)``
anyway here is the regex that worked for me
```
output_regex=r'\s*(?P<filename>.+)\s*(?P<line>\d+):(?P<column>\d+)\s*'
                  r'(\D)\s*(?P<message>.+)',
```